experimental/plugins/plugintypes/auditlog.go

```diff
@@ -31,6 +31,7 @@ type AuditLogTransaction interface {
 	Response() AuditLogTransactionResponse
 	HasResponse() bool
 	Producer() AuditLogTransactionProducer
+	HighestSeverity() string
```

Comment:
Could you add a comment on what is this? I know others don't have comments but I think since this is OCSF specific, worth checking.

Reply:
Can do.

Reply:
Comments added